examples/textual_inversion/textual_inversion.py

```diff
@@ -545,24 +551,31 @@ def main():
 
         accelerator.wait_for_everyone()
 
-    # Create the pipeline using using the trained modules and save it.
     if accelerator.is_main_process:
-        pipeline = StableDiffusionPipeline(
-            text_encoder=accelerator.unwrap_model(text_encoder),
-            vae=vae,
-            unet=unet,
-            tokenizer=tokenizer,
-            scheduler=PNDMScheduler.from_config("CompVis/stable-diffusion-v1-4", subfolder="scheduler"),
-            safety_checker=StableDiffusionSafetyChecker.from_pretrained("CompVis/stable-diffusion-safety-checker"),
-            feature_extractor=CLIPFeatureExtractor.from_pretrained("openai/clip-vit-base-patch32"),
-        )
-        pipeline.save_pretrained(args.output_dir)
-        # Also save the newly trained embeddings
+        only_save_embeds = args.only_save_embeds
+        if args.push_to_hub and args.only_save_embeds:
+            print("Enabling full model saving because --push_to_hub was specified.")
```

Comment:
```suggestion
            logger.info("Enabling full model saving because --push_to_hub was specified.")
```

Reply:
Of course. Sorry, the change just didn't make it in the last force push.